drivers/serial/uart_nrfx_uart.c

```diff
@@ -15,6 +15,7 @@
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
 static uart_irq_callback_t m_irq_callback; /**< Callback function pointer */
+static bool m_init_uart_event_txrdy;
```

Comment:
we don't use hungarian notation prefixes in Zephyr. Can you please remove the `m_` from this new variable and also from the `m_irq_callback` (in a separate commit for that one please).

Reply:
Another question, does this `bool` need to be atomic? we have `atomic_t` for this